torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -762,7 +762,6 @@ void TensorExprKernel::bindInput(const torch::jit::Value* input) {
 }
 
 TensorExprKernel::TensorExprKernel(const Node* node) {
-  KernelScope kernel_scope(kernel_arena_);
```

Comment:
I don't think the semantics is correct if we don't bind the scope when TensorExprKernel is actually used. 

You could have multiple TensorExprKernel at the same time.